lib/core/lib/ops/utils.js

```diff
@@ -13,36 +17,40 @@ function isUniqueViolationError (err) {
   return err && err.code === '23505'
 }
 
+function isForeignKeyViolationError (err) {
+  return err && err.code === '23503'
+}
+
 function checkPoliciesOrg (db, policies, organizationId, cb) {
-  db.query(SQL`SELECT id FROM policies WHERE id = ANY (${policies}) AND org_id != ${organizationId}`, (err, result) => {
+  db.query(SQL`SELECT id FROM policies WHERE id = ANY (${policies}) AND org_id = ${organizationId}`, (err, result) => {
     if (err) return cb(Boom.badImplementation(err))
 
-    if (result.rowCount !== 0) {
-      return cb(Boom.badRequest(`Some policies [${result.rows.map(r => r.id).join(',')}] were not found`))
+    if (result.rowCount !== policies.length) {
+      return cb(Boom.badRequest(`Some policies [${policies.filter(p => result.rows.map(r => r.id).indexOf(p) < 0)}] were not found`))
```

Comment:
sure, the lodash approach is always safer although I don't think it will save much